src/components/widgets-settings/width/width.directive.coffee

```diff
@@ -62,14 +62,13 @@ module.controller('SettingWidthCtrl', ($scope, $element, $timeout, $log, ImpacWi
   $scope.pdfMode = false
   ImpacDashboardsSvc.pdfModeEnabled().then(null, null, ->
     $scope.pdfMode = true
-    $scope.initiallyExpanded = $scope.expanded
     # Expand the widget if it's not already the case
     w.toggleExpanded(false) unless $scope.expanded
   )
   ImpacDashboardsSvc.pdfModeCanceled().then(null, null, ->
     $scope.pdfMode = false
     # Reduce the widget if it wasn't expanded initially
-    w.toggleExpanded(false) unless $scope.initiallyExpanded
+    w.toggleExpanded(false) unless w.initiallyExpanded
```

Comment:
@cesar-tonnoir what other scope is `w.initiallyExpanded` initialised in? Im not a big fan of these variables being attached to the widget either.. But seeing as `w.initialWidth`, and `w.width` are being initialised in the widget.directive, it makes sense to me to also initialise an `w.initiallyExpended` value. If we do pass an attribute, we will need to add that to every widget template `<settings-width>` attributes.